addons/website_sale/models/website.py

```diff
@@ -1040,3 +1035,12 @@ def _get_product_image_ratio_height(self):
             case '4_5':
                 return '96px'
         return '64px'
+
+    def _populate_product_feeds(self):
+        """Populate product feeds for the website with default values."""
+        for website in self:
+            website.env['product.feed'].create({
+                'name': website.env._("GMC 1"),
+                'website_id': website.id,
+                'lang_id': website.default_lang_id.id,
```

Comment:
why not handled through a compute method defaulting to the feed website default_lang ?